### DISCUSSION TITLE
add new widgets

### DISCUSSION THREAD

**Discussion by PharmaRacist** [end-4/dots-hyprland#411] [Q&A]
how can i add the old illogical impulse clock widgets

**Comment by end-4** [comment]
uncomment lines that say DesktopBackground in ~/.config/ags/config.js

**Reply by PharmaRacist** [reply]
bro ..... i love you man

**Comment by PharmaRacist** [comment]
how do i add gpu metrics on both bar and desktopbackground

**Reply by end-4** [reply]
it's not something you can configure simply so this is a case of "pls google it/rtfm"

**Reply by PharmaRacist** [reply]
i managed to add another disc metric and cpu the only thing i want is the command that give my the usage of amdgpu  i have googled it btw
